genomesubsampler/parseRepophlan.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017--, genome-subsampler development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# parser for RepoPhlAn-downloaded genomes
+#
+
+import click
+import pandas as pd
+
+
+def parse_repophlan(repophlan_wscores_fp):
+    """ Compute basic statistics of RepoPhlAn-downloaded genomes
```

Comment:
I think pep8 standards say that you should end all sentences with periods, thus add a .
Furthermore, save the leading whitespace. (not 100% sure if I remember those standards correctly)